src/c_kzg_4844.c

```diff
@@ -590,12 +590,13 @@ static uint32_t reverse_bits(uint32_t n) {
  * @param[in,out] values The array, which is re-ordered in-place
  * @param[in]     size   The size in bytes of an element of the array
  * @param[in]     n      The length of the array, must be a power of two
- *                       less that 2^32
+ *                       less that 2^32 and unequal to 1.
```

Comment:
I think it would be clearer to say this, since returns BADARGS for 0 too. Also fixes previous typo (that -> than).
```suggestion
 *                       greater than 1 and less than 2^32.
```

Or maybe even "between 2 and 2^32-1" but I'll leave that up to you.